src/utils/PreparePeopleHelper.ts

```diff
@@ -0,0 +1,63 @@
+import { Person } from '../types';
+import { SortParam } from '../types/SortParam';
+
+type FilterPeopleType = (
+  people: Person[],
+  gender: string | null,
+  query: string | null,
+  century: string[],
+) => Person[];
+
+export const FilterPeople: FilterPeopleType = (
```

Comment:
It's not a component, so it should have a lowerCase letter at the beginning of the name.
```suggestion
export const filterPeople: FilterPeopleType = (
```